Makefile

```diff
@@ -5,7 +5,12 @@ npm:
 node_dev:
 	echo 'Starting Node dev server in Docker Container'
 	docker build -t node_dev .
-	docker run -t -p 8080:8080 node_dev
+	docker run -it -p 8080:8080 node_dev
```

Comment:
`-it` is a common way to start containers. I believe it has to do with the container being open to inspection. Meaning, one could open a shell terminal that represents being inside the container and see what's going on.